default.env

```diff
@@ -0,0 +1,6 @@
+DATABASE_URL=postgresql://users_api:users_api@localhost/users_api
+TEST_DATABASE_URL=postgresql://users_api:users_api@localhost/test_users_api
```

Comment:
I'm guessing these settings imply that you need to have a postgresql running locally when you develop or test the application. Can you add a bit to the readme about what the setup looks like here, 1) install postgresql, 2) create a db, etc etc.